Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdb/dao/AddressDAO.java

```diff
@@ -147,69 +96,15 @@ public boolean create(Address addressRecord) {
      * @return Address
      */
     public Address read(Long id) {
-
         LOG.debug("AddressDAO.read() - Begin");
```

Comment:
Inconsistent log level. Use only one. Stick to trace level for now in this method.